README.md

```diff
@@ -69,6 +69,15 @@ In case Docker complains about not being able to connect to the Docker daemon ma
 
 After adding yourself to the `docker` group make sure to log out and back in again with your terminal.
 
+We support the following images on Docker Cloud:
+
+Name | Description
+-----|------
+`latest` | `master` compiled with release flag
+`latest-assertions` | `master` compiled with with release flag, assertions enabled and debug symbols
+`latest-debug` | `master` compiled with debug flag
+`<tag>` | specific tag compiled with release flag
+`<tag>-debug` | specific tag compiled with debug flag
```

Comment:
What would be the use case of that? Who would actually use that? Given that every build on Docker Cloud takes 1 hour (and they are currently not parallelized, so each commit is 3h), I'm hesitant to add more.